onboard/cv/main.cpp

```diff
@@ -163,13 +163,12 @@ int main() {
       obstacle_return obstacle_detection =  avoid_obstacle_sliding_window(depth_img, src,  num_sliding_windows , roverPixWidth);
       if(obstacle_detection.bearing > 0.05 || obstacle_detection.bearing < -0.05) {
         // cout<< "bearing not zero!\n";
-        obstacleMessage.detected = true;    //if an obstacle is detected in front
         obstacleMessage.distance = obstacle_detection.center_distance; //update LCM distance field
       }
       obstacleMessage.bearing = obstacle_detection.bearing;
 
       #if PERCEPTION_DEBUG
-      // cout << "Turn " << obstacleMessage.bearing << ", detected " << (bool)obstacleMessage.detected<< endl;
+      // cout << "Turn " << obstacleMessage.bearing << ", distance " << (bool)obstacleMessage.distance << endl;
       #endif
```

Comment:
Should we just remove this code since it doesn't appear to be doing anything?

Reply:
since it's for debugging I wanted to leave the framework there so that I don't change more than necessary in perceptions's code, especially following the new workflow

Reply:
Your change is affecting this code so you should be responsible for this. My point was that it doesn't make sense to both have an `#if` and be commented out. This implies that the `#if` isn't working and can be removed or the commented out code is not wanted. You should talk to @mmartin4972 about a plan forward for this. If this is the extent of the problem, you should fix it. If it is a larger issue in `onboard/cv`, you should create an issue to fix this since you have exposed a problem.

Reply:
#428 issue created